src/app/core/admin/admin-entity-details/admin-entity-field/configure-entity-field-validator/configure-entity-field-validator.component.html

```diff
@@ -0,0 +1,33 @@
+<form class="validator-form-cell flex-column" [formGroup]="form">
+  <mat-tab-group formGroupName="validatorForm">
```

Comment:
my IDE remarks that mat-tab-group is not allowed here (must be within the template that has the mat-tab --> take out of the template here and wrap in admin-entity-field instead)